### PR TITLE
MAT-236

### PR DESCRIPTION
**Pull request by scarabus**

### CODE REVIEW

**Comment by amkimian** [comment]
Can there be a unit test added to prove this change?

**Comment by scarabus** [comment]
There is a test in https://github.com/Incapture/Matrix/pull/206 - I can add one here too.

**Comment by jonathan-major** [comment]
rfx cast test failed http://52.53.235.94/job/Rapture_OS_PR_Builder/86/testReport/junit/reflex/CastTest/testCast1/

File: Libs/RaptureCommon/src/test/java/rapture/common/RaptureJsonSerializerTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        int ilval = 1000000000;
+
+        @JsonProperty
+        @JsonSerialize(using = RaptureJsonSerializer.class)
+        Integer Iilval = 1000000000;
+
+        @JsonSerialize(using = RaptureJsonSerializer.class)
```
Comment (amkimian) [inline Libs/RaptureCommon/src/test/java/rapture/common/RaptureJsonSerializerTest.java]: Do we have to add this annotation to any class/object that needs to have this scientific notation disabled?

Comment (scarabus) [inline Libs/RaptureCommon/src/test/java/rapture/common/RaptureJsonSerializerTest.java]: Only for Doubles and Floats.
I haven't found a solution that works for primitives.
Integer values don't seem to need it. 
BigDecimals are handled separately.

**Pull request merged**